ietf/meeting/models.py

```diff
@@ -1379,4 +1379,11 @@ class MeetingHost(models.Model):
 
     class Meta:
         unique_together = (('meeting', 'name'),)
-        ordering = ('pk',)
\ No newline at end of file
+        ordering = ('pk',)
+
+class Attended(models.Model):
+    person = ForeignKey(Person)
```

Comment:
Should there be a `unique_together` constraint on `('person', 'session')` to prevent the possibility of someone attending a session twice?

Reply:
yes, added.